index.html

```diff
@@ -97,86 +96,156 @@ <h1>bam2.iobio</h1>
+            <iobio-info-button label="Singletons">
+              <div slot="header">
+                <h4>Singletons</h4>
+              </div>
+              <div slot="content">
+                <p>This is custom content for the Singletons percent box.</p>
+              </div>
+            </iobio-info-button>
+
+            <iobio-percent-box percent-key="singletons" total-key="total_reads"></iobio-percent-box>
           </iobio-panel>
 
           <iobio-panel>
-            <iobio-percent-box
-              label="Both Mates Mapped"
-              percent-key="both_mates_mapped"
-              total-key="total_reads">
-            </iobio-percent-box>
+            <iobio-info-button label="Both Mates Mapped">
+              <div slot="header">
+                <h4>Both Mates Mapped</h4>
+              </div>
+              <div slot="content">
+                <p>This is custom content for the Both Mates Mapped percent box.</p>
+              </div>
+            </iobio-info-button>
+
+            <iobio-percent-box percent-key="both_mates_mapped" total-key="total_reads"></iobio-percent-box>
           </iobio-panel>
          
           <iobio-panel>
-            <iobio-percent-box
-              label="Duplicates"
-              percent-key="duplicates"
-              total-key="total_reads">
-            </iobio-percent-box>
+            <iobio-info-button label="Duplicates">
+              <div slot="header">
+                <h4>Duplicates</h4>
+              </div>
+              <div slot="content">
+                <p>This is custom content for the Duplicates percent box.</p>
+              </div>
+            </iobio-info-button>
+
+            <iobio-percent-box percent-key="duplicates" total-key="total_reads"></iobio-percent-box>
           </iobio-panel>
 
         </div>
 
         <div class='histogram-column'>
           <iobio-panel>
-            <iobio-histogram
-              label="Read Coverage Distribution"
-              broker-key="coverage_hist">
-            </iobio-histogram>
+            <iobio-info-button label="Read Coverage Distribution">
+              <div slot="header">
+                <h4>Read Coverage Distribution</h4>
+              </div>
+              <div slot="content">
+                <p>This is custom content for the Read Coverage Distribution histogram.</p>
+              </div>
+            </iobio-info-button>
+
+            <iobio-histogram broker-key="coverage_hist"></iobio-histogram>
           </iobio-panel>
-
+          
           <iobio-panel>
-            <iobio-tabs label-1="Fragment Length" label-2="Read Length">
-              <iobio-histogram
-                broker-key="frag_hist"
-                ignore-outliers>
-              </iobio-histogram>
-
-              <iobio-histogram
-                broker-key="length_hist"
-                ignore-outliers>
-              </iobio-histogram>
+            <iobio-tabs>
+              <iobio-info-button slot="info-button-1">
```

Comment:
Is there a way to avoid having "info-button-1", "info-button-2", etc? For example, what if I had 10 tabs? Can we just move this into <iobio-tab> and then let the developer put whatever content they want? So in our case we would put a <iobio-info-button> inside the <iobio-tab>. It's ok if it doesn't look exactly like bam1, but if you want that you could also add an attribute to <iobio-info-button> that lets you choose whether to put the icon to the left or right of the header text.